documentation/assemblies/security/assembly-security.adoc

```diff
@@ -3,7 +3,7 @@
 // master.adoc
 
 [id='security-{context}']
-= Security
+= Managing TLS certificates for secure communication
```

Comment:
I think this title is a bit too long when compared to the others. The chapter is a mixed bag; there are discussions of Secrets and keys as well as TLS certificates.

Alternatives to consider:

TLS encryption and authentication

TLS encryption

TLS and certificate management

Reply:
Thanks Daniel. It's all about maintaining certificates really, so I thought _Maintaining TLS certificates_